storage/src/adb/any/variable/mod.rs

```diff
@@ -608,6 +608,22 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Codec, H: CHasher, T: Translato
         Ok(())
     }
 
+    /// Get the metadata associated with the last commit, or None if no commit has been made.
+    pub async fn get_metadata(&self) -> Result<Option<V>, Error> {
```

Comment:
nit: Let's return the `op` index of the last commit inside of `Option` (so `Option<u64, V>`)